osconfig_tests/test_suites/inventory/inventory.go

```diff
@@ -62,6 +62,19 @@ gpgkey=https://packages.cloud.google.com/yum/doc/yum-key.gpg
 EOM
 yum -y install google-osconfig-agent
 echo 'inventory install done'`
+var installInventoryYumEL6 = `sleep 10
```

Comment:
Yeah, this test specifically is just for inventory so that why i used this variable name here, the agent is the osconfig agent that will handle a number of different scenarios. I renamed to avoid this confusion